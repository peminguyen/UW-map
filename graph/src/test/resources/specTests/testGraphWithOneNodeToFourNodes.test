CreateGraph graphWithOneNodeToFourNodes

AddNode graphWithOneNodeToFourNodes A
AddNode graphWithOneNodeToFourNodes B
AddNode graphWithOneNodeToFourNodes C
AddNode graphWithOneNodeToFourNodes D
AddNode graphWithOneNodeToFourNodes E

AddEdge graphWithOneNodeToFourNodes A B eAB
AddEdge graphWithOneNodeToFourNodes A C eAC
AddEdge graphWithOneNodeToFourNodes A D eAD
AddEdge graphWithOneNodeToFourNodes A E eAE

ListNodes graphWithOneNodeToFourNodes
ListChildren graphWithOneNodeToFourNodes A
ListChildren graphWithOneNodeToFourNodes B
ListChildren graphWithOneNodeToFourNodes C
ListChildren graphWithOneNodeToFourNodes D
ListChildren graphWithOneNodeToFourNodes E