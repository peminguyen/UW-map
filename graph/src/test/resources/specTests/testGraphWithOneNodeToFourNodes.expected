created graph graphWithOneNodeToFourNodes

added node A to graphWithOneNodeToFourNodes
added node B to graphWithOneNodeToFourNodes
added node C to graphWithOneNodeToFourNodes
added node D to graphWithOneNodeToFourNodes
added node E to graphWithOneNodeToFourNodes

added edge eAB from A to B in graphWithOneNodeToFourNodes
added edge eAC from A to C in graphWithOneNodeToFourNodes
added edge eAD from A to D in graphWithOneNodeToFourNodes
added edge eAE from A to E in graphWithOneNodeToFourNodes

graphWithOneNodeToFourNodes contains: A B C D E
the children of A in graphWithOneNodeToFourNodes are: B(eAB) C(eAC) D(eAD) E(eAE)
the children of B in graphWithOneNodeToFourNodes are:
the children of C in graphWithOneNodeToFourNodes are:
the children of D in graphWithOneNodeToFourNodes are:
the children of E in graphWithOneNodeToFourNodes are: